scylla/src/response/query_result.rs

```diff
@@ -444,6 +458,1180 @@ impl QueryRowsResult {
 
         (raw_rows_with_metadata, tracing_id, warnings)
     }
+
+    pub fn rows_displayer(&self) -> RowsDisplayer<'_> {
+        RowsDisplayer::new(self)
+    }
+}
+
+/// A utility struct for displaying rows received from the database in a [`QueryRowsResult`].
+///
+/// This struct provides methods to configure the display settings and format the rows
+/// as a table for easier visualization. It supports various display settings such as
+/// terminal width, color usage, and formatting of different CQL value types.
+///
+/// # Example
+///
+/// ```rust
+/// # use scylla::response::query_result::{QueryResult, QueryRowsResult, RowsDisplayer};
+/// # fn example(query_result: QueryResult) -> Result<(), Box<dyn std::error::Error>> {
+///     let rows_result = query_result.into_rows_result()?;
+///     let mut displayer = RowsDisplayer::new(&rows_result);
```

Comment:
:wrench: Let's suggest `rows_displayer` as the default way to create a `RowsDisplayer`.
```suggestion
///     let rows_result = query_result.into_rows_result()?;
///     let mut displayer = rows_result.rows_displayer();
```